README.md

```diff
@@ -26,20 +26,28 @@ An open source web-based comic book reader.
 $ yarn add villain-react
 ```
 
+## Development
+
+- Run `yarn install` in the console
```

Comment:
I think should be just `yarn` not sure if there is any significant difference but is more simple :smile: